catalogue/wcrs/documents-advanced-search/api.yml

```diff
@@ -1,12 +1,14 @@
-title: Documents Advanced Search
-description: Perform faceted searches against the content and metadata of patient documents
-type: Solr 
+title: Patient document advanced search
+description:   Perform an advanced search of WCRS to find patient documents using Apache Solr. Group and filter results by category to find documents more quickly.
+type: Solr
 owner: NWIS - Corporate Applications
 status: stable
 access: restricted
 overview: |
-  We use Apache Solr to prvoide advanced search against patient documents in the Welsh Care Record Service.
+  Perform an advanced search of WCRS to find patient documents using Apache Solr.
+  Group and filter results by category to find documents more quickly.
+  We use Apache Solr to provide advanced search against patient documents in the Welsh Care Record Service.
```

Comment:
repetition (also as discussed "Apache Solr faceted search" could be considered a key word/phrase for describing the API)